recipe/meta.yaml

```diff
@@ -5,31 +5,51 @@ package:
   version: {{ version }}
 
 source:
-  url: ftp://ftp.info-zip.org/pub/infozip/src/zip{{ version.replace(".", "") }}.tgz
-  sha256: f0e8bb1f9b7eb0b01285495a2699df3a4b766784c1765a8f1aeedf63c0806369
+  # patches as applied in
+  # https://gitlab.archlinux.org/archlinux/packaging/packages/zip/-/blob/main/PKGBUILD?ref_type=heads
+  - url: https://src.fedoraproject.org/rpms/zip/raw/rawhide/f/zip-3.0-currdir.patch
+    sha256: 44cac3d4e8b447988b6e4eeb4bb6e701110cedfb09ea49eac471011328c8acab
+  - url: https://src.fedoraproject.org/rpms/zip/raw/rawhide/f/zip-3.0-exec-shield.patch
+    sha256: 10f0f3756a2415e33edf1455373b322404962b743b3a52a757b669e47dd7f1b2
+  - url: https://src.fedoraproject.org/rpms/zip/raw/rawhide/f/zip-3.0-format-security.patch
+    sha256: 3759134487afad9c63e2e9693aa05b9dcc67f55a9ef961e82074b587b094292a
+  - url: https://src.fedoraproject.org/rpms/zip/raw/rawhide/f/zipnote.patch
+    sha256: 89249a3f1fdf838b795ce432a2d763fdbe913d6a146541e41b7a2e2769291ba0
+  - url: https://downloads.sourceforge.net/infozip/zip{{ version.replace(".", "") }}.tar.gz
+    sha256: f0e8bb1f9b7eb0b01285495a2699df3a4b766784c1765a8f1aeedf63c0806369
+    patches:
+      - "{{ SRC_DIR }}/zip-3.0-currdir.patch"
+      - "{{ SRC_DIR }}/zip-3.0-exec-shield.patch"
+      - "{{ SRC_DIR }}/zip-3.0-format-security.patch"
+      - "{{ SRC_DIR }}/zipnote.patch"
 
```

Comment:
I didn't know this syntax worked. cool!

Reply:
It was a hunch! With some of this stuff, found it can be better to be _lucky and happy_ than _sure and sad_!

Reply:
nice attitude!

Reply:
scrambling up the shoulders of giants. i'm glad sibling (and twice sibling, in this case) stuff have solved most of these things... it's really just when we make our own problems... like try to run stuff on windows :stuck_out_tongue_closed_eyes: